### ISSUE TITLE
Microsoft Teams integration

### ISSUE THREAD

**Issue by vasjapavlovic** [O365/python-o365#378]
When do you plan to integrate microsoft teams in the module?
I try to create a channel in a team. can you point out how to achive that?

Endpoint:
`https://graph.microsoft.com/v1.0/teams/{team-id}/channels`

Data:

```
{
  "displayName": "Architecture Discussion",
  "description": "This channel is where we debate all future architecture plans"
}
```

**Comment by janscas** [comment]
There's currently no plans to add Microsoft Teams support.

**Comment by matt43121** [comment]
Hi,

Are you accepting pull requests on this? I have some code based on your implementation of the planner resource if you are that may satisfy this need.

**Comment by janscas** [comment]
Of course. It just needs to comply with the library code style and philosophy.

**Comment by matt43121** [comment]
@janscas Just looking back at this. In order to expand teams integration, most of the base of MS Teams are on the groups/users endpoints. Are you wanting the users and group endpoints to be in unique files separate from the Teams code or not bothered?

**Comment by janscas** [comment]
Hi @matt43121 
Thanks for asking

The [User ](https://github.com/O365/python-o365/blob/master/O365/directory.py#L12)object is already done (lacking some methods) in the directory.py file.

I think Groups should have their own file because is a separate functionality (but related to teams and more).

**Comment by tomiszili** [comment]
Hi all!

Could anybody help me, i dind't find any example or documentation about this [problem](https://github.com/O365/python-o365/issues/429#issue-605500797)?

Thanks!

**Comment by alejcas** [comment]
Teams was added later on
